.freeipa-pr-ci.yaml

```diff
@@ -62,3 +62,123 @@ jobs:
         template: *ci-master-f27
         timeout: 3600
         topology: *master_1repl
+
+  fedora-27/test_topologies:
+    requires: [fedora-27/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{fedora-27/build_url}'
+        test_suite: test_integration/test_topologies.py
+        template: *ci-master-f27
+        timeout: 3600
+        topology: *master_1repl
+
+  fedora-27/test_sudo:
+    requires: [fedora-27/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{fedora-27/build_url}'
+        test_suite: test_integration/test_sudo.py
+        template: *ci-master-f27
+        timeout: 3600
+        topology: *master_1repl_1client
+
+  fedora-27/test_kerberos_flags:
+    requires: [fedora-27/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{fedora-27/build_url}'
+        test_suite: test_integration/test_kerberos_flags.py
+        template: *ci-master-f27
+        timeout: 3600
+        topology: *master_1repl_1client
+
+  fedora-27/test_http_kdc_proxy:
+    requires: [fedora-27/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{fedora-27/build_url}'
+        test_suite: test_integration/test_http_kdc_proxy.py
+        template: *ci-master-f27
+        timeout: 3600
+        topology: *master_1repl_1client
```

Comment:
This test needs only 1 client and 0 replica

Reply:
Please see my comment below.